ui/src/containers/NodeInformation.js

```diff
@@ -10,29 +10,29 @@ import { fetchPodsAction } from '../ducks/app/pods';
 import { refreshNodesAction, stopRefreshNodesAction } from '../ducks/app/nodes';
 import {
   fetchVolumesAction,
-  fetchPersistentVolumeAction
+  fetchPersistentVolumeAction,
```

Comment:
I think all those trailing commas were added by the new linter settings Patrick added recently.

Ideally they would be in a dedicated commit:
- one commit for the linting/auto formatting
- one commit for your feature

Would make the review easier (you can blind approve the linter commit, and read the feature commit carefully without the noise).

Reply:
I can provide Git guidance to do so if needed :slightly_smiling_face: